lib/FrameManager.js

```diff
@@ -464,9 +464,11 @@ class Frame {
      * @param {string} url
      * @return {!Promise<!HTMLElement>}
      */
-    async function addScriptUrl(url) {
+    async function addScriptUrl(url, type) {
       const script = document.createElement('script');
       script.src = url;
+      if (type === 'module')
```

Comment:
should we pass through any type if the user passes one in? e.g. allow someone to inject a HB template: `<script type="text/x-handlebars-template"`.